integration-tests/src/test/java/org/apache/druid/tests/coordinator/duty/ITAutoCompactionTest.java

```diff
@@ -456,7 +466,7 @@ public void testAutoCompactionDutySubmitAndVerifyCompaction() throws Exception
       verifySegmentsCount(4);
       verifyQuery(INDEX_QUERIES_RESOURCE);
 
-      submitCompactionConfig(MAX_ROWS_PER_SEGMENT_COMPACTED, Period.days(1));
+      submitCompactionConfig(MAX_ROWS_PER_SEGMENT_COMPACTED, Period.days(1), null);
```

Comment:
It is better to retain the old signature of `submitCompactionConfig` which doesn't require an engine rather than having to pass null everywhere.

Reply:
Have created some more overloaded functions to avoid the additional null arg.